planex/cmd/makesrpm.py

```diff
@@ -127,11 +127,10 @@ def populate_working_directory(tmpdir, spec):
     """
 
     sources = [os.path.basename(source[0]) for source in spec.sources()]
-    for source in sources:
-        try:
-            spec.extract_source(source, tmpdir)
-        except KeyError:
-            print("Could not find a source for %s" % source)
+    try:
+        spec.extract_sources(sources, tmpdir)
+    except KeyError as err:
+        print("Could not find a source for %s" % err.message)
```

Comment:
`BaseException.message` is long deprecated, this should be `str(e)`

http://sayspy.blogspot.co.uk/2007/05/baseexceptionmessage-is-now-deprecated.html